tripal_chado/src/Plugin/Field/FieldType/ChadoContactTypeDefault.php

```diff
@@ -162,20 +162,23 @@ public static function tripalTypes($field_definition) {
     // An intermediate linker table is used
     else {
       // Define the linker table that links the base table to the object table.
+      // E.g.,:  project_contact.project_contact_id
       $properties[] = new ChadoIntStoragePropertyType($entity_type_id, self::$id, 'linker_id', $record_id_term, [
         'action' => 'store_pkey',
         'drupal_store' => TRUE,
         'path' => $linker_table . '.' . $linker_pkey_col,
       ]);
 
       // Define the link between the base table and the linker table.
+      // E.g.,:  project.project_id>project_contact.project_id
       $properties[] = new ChadoIntStoragePropertyType($entity_type_id, self::$id, 'link', $linker_left_term, [
         'action' => 'store_link',
         'drupal_store' => TRUE,
         'path' => $base_table . '.' . $base_pkey_col . '>' . $linker_table . '.' . $linker_left_col,
       ]);
 
       // Define the link between the linker table and the object table.
+      // E.g.,:  project_contact.contact_id
```

Comment:
```suggestion
      // E.g.:  project_contact.project_contact_id
      $properties[] = new ChadoIntStoragePropertyType($entity_type_id, self::$id, 'linker_id', $record_id_term, [
        'action' => 'store_pkey',
        'drupal_store' => TRUE,
        'path' => $linker_table . '.' . $linker_pkey_col,
      ]);

      // Define the link between the base table and the linker table.
      // E.g.:  project.project_id>project_contact.project_id
      $properties[] = new ChadoIntStoragePropertyType($entity_type_id, self::$id, 'link', $linker_left_term, [
        'action' => 'store_link',
        'drupal_store' => TRUE,
        'path' => $base_table . '.' . $base_pkey_col . '>' . $linker_table . '.' . $linker_left_col,
      ]);

      // Define the link between the linker table and the object table.
      // E.g.:  project_contact.contact_id
```
LOL I feel like there is too much punctuation 😁